packages/charting/src/index.ts

```diff
@@ -1,2 +1,3 @@
-export { LineChart } from './LineChart';
-export { VerticalBarChart } from './VerticalBarChart';
+export * from './LineChart';
+export * from './VerticalBarChart';
+export * from './HorizontalBarChart';
```

Comment:
How did you even discover this... ? This is a very thorough audit :)

Reply:
I tried fixing the examples to import from the package, and they couldn't do it without a file based import. :)